src/features/dashboard/components/ConvCardList.tsx

```
@@ -0,0 +1,10 @@
+import { useGetAllConversations } from "@/features/conversation/api/get-all-conversations";
+import React from "react";
+
+const ConvCardList = () => {
+  const { data, isLoading } = useGetAllConversations();
```

ESLint

[error] 'isLoading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)